assignments/homework/chernyavskaya/hw2.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+from astropy import units as u
+import astropy.constants as const
+from astropy.io import fits
+from astropy.table import Table, QTable
+from astropy.time import Time
+from astropy.coordinates import SkyCoord
+
+import matplotlib.pyplot as plt
+plt.rc('image', origin='lower')
+plt.rc('figure', figsize=(10, 6))
+
+# Challenge
+#1. Examine the headers of the first and second HDU in the point source catalog and try adding new keywords to them or changing them
+hdu1 = fits.open('astropy_notebooks/data/LMCDensFits1k.fits')
```

Comment:
The file paths are not pointing to the right place relative to this homework file. 
```suggestion
hdu1 = fits.open('../../../astropy_notebooks/data/LMCDensFits1k.fits')
```